packages/core-mobile/app/hooks/earn/useAdvancedSearchNodes.ts

```diff
@@ -40,16 +42,30 @@ export const useAdvancedSearchNodes = ({
   validators,
   sortFilter,
   searchText
-}: useAdvancedSearchNodesProps) => {
+}: useAdvancedSearchNodesProps):
+  | {
+      validators: NodeValidators | undefined
+      error: undefined
+    }
+  | {
+      validators: never[]
+      error: Error
+    } => {
   const { data: peers } = usePeers()
-
+  const activeNetwork = useSelector(selectActiveNetwork)
   const isDeveloperMode = useSelector(selectIsDeveloperMode)
   const isEndTimeOverOneYear = isOverOneYear(stakingEndTime)
   const noMatchError = new Error(
     `no node matches filter criteria: stakingAmount:  ${stakingAmount}, stakingEndTime: ${stakingEndTime}, minUpTime: ${minUpTime}`
   )
   const noValidatorsError = new Error(`no validators found.`)
 
+  // TODO: remove this after Etna activation
```

Comment:
I just created this ticket https://ava-labs.atlassian.net/browse/CP-9539. can you add it to the comment?